src/test/java/net/snowflake/client/jdbc/ResultSetArrowForceTZMultiTimeZoneIT.java

```diff
@@ -65,17 +66,23 @@ private void testTimestampTZWithScale(int scale) throws SQLException {
     String column = "(a timestamp_tz(" + scale + "))";
 
     String values = "('" + StringUtils.join(cases, "'),('") + "'), (null)";
-    Connection con = init(table, column, values);
-    ResultSet rs = con.createStatement().executeQuery("select * from " + table);
-    int i = 0;
-    while (i < cases.length) {
-      rs.next();
-      assertEquals(times[i++], rs.getTimestamp(1).getTime());
-      assertEquals(0, rs.getTimestamp(1).getNanos());
+    try (Connection con = init(table, column, values);
+        Statement statement = con.createStatement();
+        ResultSet rs = statement.executeQuery("select * from " + table)) {
+      try {
+        int i = 0;
+        while (i < cases.length) {
+          rs.next();
```

Comment:
let's wrap `rs.next()` with `assertTrue(...)` in all changed tests

Reply:
I was applied to all the rs.next(). FYI, some rs.next () is used to check that the result set is at isAfterLast status, so the rs.next() returns false in this case. As a result, I applied assertFalse to that testing.